.gitignore

```diff
@@ -3,6 +3,7 @@ coverage.txt
 .terraform
 .terraform.lock.hcl
 /cmd/terramate/terramate
+/testdata/testserver/my-user.tm
```

Comment:
Is this a file that's generated during tests? 🤔 
If this is a one off file that you needed for manual tested, I'd suggest using the global ~/.gitignore instead. You can activate it by doing:

```
git config --global core.excludesFile '~/.gitignore'
```

Reply:
This is the file where I configure my user to the default organization. I added it to the gitignore because I was going to show my approach and then others could do the same.